types/cucumber/index.d.ts

```diff
@@ -1,11 +1,11 @@
-// Type definitions for cucumber-js 2.1
+// Type definitions for cucumber-js 3.1
 // Project: https://github.com/cucumber/cucumber-js
 // Definitions by: Abraão Alves <https://github.com/abraaoalves>
 //                 Jan Molak <https://github.com/jan-molak>
 //                 Isaiah Soung <https://github.com/isoung>
 //                 BendingBender <https://github.com/BendingBender>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.4
```

Comment:
It doesn't look like you're adding anything that requires a minimum TS version of 2.4. Could you revert this?

Reply:
:eyes:

Reply:
Will do tomorrow, and you're right. My bad.

Reply:
Opened a PR on your fork @ath88!
Ran the tests and such as well. 

If you have time today, that would be _sweet_

https://github.com/ath88/DefinitelyTyped/pull/1

Reply:
Thanks! Done.

Reply:
No problem :smile_cat: Thanks for checking it in!